src/js/routes/services.js

```diff
@@ -13,7 +13,7 @@ let serviceRoutes = {
   children: [
     {
       type: Route,
-      name: 'services-deployments',
+      name: '-services-deployments',
```

Comment:
I am late to the game here, my apologies.
I know that this will eventually be removed from react-router, but this hack to differentiate the top level route is not very nice. At least it got me confused and puzzled for some time until I started inspecting it in depth, which is not favourable. 
I believe we should stay consistent (and it shouldn't be too big of a change) with how we do it in the nodes page and name it `service-page`, even if it requires updating routes around the applicaiton: https://github.com/dcos/dcos-ui/blob/master/src/js/routes/nodes.js#L16